sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryInit.android.kt

```diff
@@ -11,10 +11,18 @@ import io.sentry.kotlin.multiplatform.extensions.toAndroidSentryOptionsCallback
 internal actual fun initSentry(configuration: OptionsConfiguration) {
     val options = SentryOptions()
     configuration.invoke(options)
-    SentryAndroid.init(applicationContext, options.toAndroidSentryOptionsCallback())
+
+    val context = applicationContext ?: run {
+        // TODO: add logging later
+        return
```

Comment:
tbh, I am a bit hesitant with this approach, because if someone removes our `SentryContextProvider` for whatever reason (as mentioned [here](https://docs.sentry.io/platforms/android/configuration/manual-init/#manual-initialization) for example), the SDK init will silently fail and the users wont be aware of that. Perhaps there's a better way to handle ProcessPhoenix specifically?

Reply:
is the problem that ContentProviders are not loaded at all, or that `context` or `context.applicationContext` is null?

Reply:
I've set up some debug logs and the application init is triggered twice and during the the content provider is not loaded before so applicationContext crashes due to `lateinit` but on the second load it works fine because the content provider has already set the context:

```
Rebirth triggered now (click rebirth button)
Try init sentry android: app context null (app tries to restart process, content provider is not triggered before)
Content provider loaded (now content provider is triggered)
Try init sentry android: app context sample.kmp.app.android.SentryApplication@d0c4ab0 (now we have access to app context)
```

Reply:
ok, I see, I guess there's no many ways around it anyway. After we implement logging that should help also 👍